### PR TITLE
Force language

### PR DESCRIPTION
**Pull request by MariusBluem**
* will be added with https://github.com/nextcloud/server/pull/5498
* please backport to `stable12` as soon as https://github.com/nextcloud/server/pull/5509 was merged.

Signed-off-by: Marius Blüm <marius@lineone.io>

### CODE REVIEW

File: admin_manual/configuration_server/language_configuration.rst [hunk 1]
```diff
@@ -1,24 +1,41 @@
...
+----------------
+
 In normal cases Nextcloud will automatically detect the language of the Web-GUI.
 If this does not work properly or you want to make sure that Nextcloud always
-starts with a given language, you can use the **default_language** parameter.
+starts with a given language, you can set a **default_language** parameter in the
+:file:`config/config.php`.
+
+.. note:: The default_language paramenter is only used, when the browser does not 
+   send any language, and the user did not configured own language preferences.
```
Comment (nickvergessen) [inline admin_manual/configuration_server/language_configuration.rst]: *configure, the past tense is already in `did`

File: admin_manual/configuration_server/language_configuration.rst [hunk 2]
```diff
@@ -1,24 +1,41 @@
...
-codes.
+    "default_language" => "en",
 
-Parameters
-----------
+
+Force language
+--------------
+
+If you want to force a specific language, users will not longer be able to change
```
Comment (nickvergessen) [inline admin_manual/configuration_server/language_configuration.rst]: \*no longer

File: admin_manual/configuration_server/language_configuration.rst (hunk 1 above)
Comment (nickvergessen) [inline admin_manual/configuration_server/language_configuration.rst]: Also I'd say `did not configure their own language`

Comment (pixelipo) [inline admin_manual/configuration_server/language_configuration.rst]: `hasn't` ?

**Comment by MorrisJobke** [comment]
Fixed both and now looks good

**Comment by MorrisJobke** [comment]
stable12 0c905f69

**Pull request merged**